docs/en/latest/plugins/redirect.md

```diff
@@ -124,6 +124,17 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
+Testing based on the above examples :
+
+```shell
+$ curl http://127.0.0.1:9080/hello -i
+HTTP/1.1 301 Moved Permanently
+...
+Location: https://127.0.0.1/hello
```

Comment:
Since we use `9080` to accept HTTP traffic, should we use `9443` here?

Reply:
Why should be `9443`? and the default port of `https` is `443`.
When `http_to_https` is set to true and the request is HTTP, it will be automatically redirected to HTTPS with 301 response code, and the URI will keep the same as client request.

https://github.com/apache/apisix/blob/9b98f1d13187c58195543bd48cb8b66e33aeab27/apisix/plugins/redirect.lua#L155-L166

Reply:
> Why should be `9443`

The default port for the APISIX proxy https protocol is 9443.

Reply:
@mangoGoForward Just for the consistence of the documentation, look through the whole doc you'll find we don't use the default `80` port.

Reply:
> The default port for the APISIX proxy https protocol is 9443.

@tokers See the `Location` header of response, it redirected to `https`. And I found some test cases in `t/plugin/redirect.t`:
https://github.com/apache/apisix/blob/42e84e437fa532f24523f5b38015436e1fcc013f/t/plugin/redirect.t#L431-L438

Seems we don't need to update?

Reply:
Personally I think there is a bug, The generated Location header should respect the setting of SSL port, we cannot assume the SSL port is `443`.

Also, my original meaning is quite simple, just because we use `9080` as the sample port to accept HTTP traffic, for the sake for consistence, I think using `9443` is better.

Reply:
Sorry, I don't know how to use `9443` in this test case.

Do we need create a upstream?

Reply:
This PR does not need to be modified in the test case to 9443, just use 9443 in the documentation? cc @tokers

Reply:
Ping @tokers

Reply:
@tzssangglass is correct, just use `9443` in documentation is enough.